src/saturn/game.c

```diff
@@ -597,15 +597,16 @@ void MoveEntity(Entity* entity) {
 }
 
 void func_06079BB4(s32* param_1) {
-    s32* temp = *param_1;
+    s32* temp = (s32*)*param_1;
```

Comment:
param_1 is most likely Entity* and this is doing `self->unk0`;

Reply:
I didn't decompile that function, you did, i just put the casting there to avoid the warning.